concepticondata/conceptlists/Heeschen-1978-104.tsv

```diff
@@ -0,0 +1,106 @@
+ID	NUMBER	ENGLISH	CONCEPTICON_ID	CONCEPTICON_GLOSS
+Heeschen-1978-1	1	all	98	ALL
+Heeschen-1978-2	2	ashes	646	ASH
+Heeschen-1978-3	3	bark	1204	BARK
+Heeschen-1978-4	4	belly	1251	BELLY
+Heeschen-1978-5	5	big	1202	BIG
+Heeschen-1978-6	6	bird	937	BIRD
+Heeschen-1978-7	7	let us bite	1403	BITE
+Heeschen-1978-8	8	black	163	BLACK
+Heeschen-1978-9	9	blood	946	BLOOD
+Heeschen-1978-10	10	bone	1394	BONE
+Heeschen-1978-11	11	breast	1402	BREAST
+Heeschen-1978-12	12	let us burn	2102	BURN
+Heeschen-1978-13	13	claw	72	CLAW
+Heeschen-1978-14	14	cloud	1489	CLOUD
+Heeschen-1978-15	15	cold	1287	COLD
+Heeschen-1978-16	16	come!	1446	COME
+Heeschen-1978-17	17	died	2101	DIE OR BE DEAD
+Heeschen-1978-18	18	dog	2009	DOG
+Heeschen-1978-19	19	word, language	1307	LANGUAGE
```

Comment:
last point of critics: given the frequency of this metaphor of word and language in languages of the world, making a meta-concept "LANGUAGE OR WORD" and adding relations to LANGUAGE / 1307 as well as WORD / 1599 seems also as a good option, but not necessarily needed at this point.